plugins/GSdx/GSRenderer.cpp

```diff
@@ -83,6 +83,27 @@ void GSRenderer::ResetDevice()
     if(m_dev) m_dev->Reset(1, 1);
 }
 
+// Mizuiro constantly alters it's DH value on the DISP register from 464 to 480
+// This causes the screen to shake vertically due to the frequent change in DISP register value
+// This doesn't happen in the PS2, the assumption is that the GS has a delay before applying
+// a new DH value to the output circuit. The following function handles such delay.
+GSVector4i DelayDisplayRectangleChange(const GSVector4i& current, const GSVector4i& previous, int context)
+{
+	if (previous.eq(GSVector4i(0)))
+		return current;
+
+	static std::array<bool, 2> initialized_delay = { false, false };
+	if (!initialized_delay[context] && current.x == previous.x && current.y == previous.y && current.w != previous.w && current.z == previous.z)
+	{
+		GL_PUSH("Delay on display rectangle height transistion (Requesting change from %d to %d) ", previous.w, current.w);
```

Comment:
`GL_PUSH` allow to group log. So instead to have a flat log, you can have a tree. So `GL_PUSH` is nice for big operation that will generate various API calls. In your case, you just need a standard log (for example `GL_INS` if my memory is right). Check `stdafx.h`